build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -81,10 +81,16 @@
         <SharedFramework Remove="*" />
         <SharedFramework Include="$(SharedFrameworkPublishDirectory)/**/*" />
 
-        <AssetsToRemoveFromDeps Remove="*" />
-        <AssetsToRemoveFromDeps Include="csc" />
-        <AssetsToRemoveFromDeps Include="vbc" />
-        <AssetsToRemoveFromDeps Include="MSBuild" />
+        <RuntimeTargetsAssetsToRemoveFromDeps Remove="*" />
+        <RuntimeTargetsAssetsToRemoveFromDeps Include="csc" />
+        <RuntimeTargetsAssetsToRemoveFromDeps Include="vbc" />
+        <RuntimeTargetsAssetsToRemoveFromDeps Include="MSBuild" />
+
+        <RuntimeAssetsToRemoveFromDeps Remove="*" />
+        <RuntimeAssetsToRemoveFromDeps Include="redist" />
+        <RuntimeAssetsToRemoveFromDeps Include="tool_csc" />
```

Comment:
This specific line is probably no longer necessary since this code is all in tool_roslyn.csproj.

---
In reply to: [97906067](https://github.com/dotnet/cli/pull/5415#discussion_r97906067) [](ancestors = 97906067)